src/OrbitGl/GpuTrack.h

```diff
@@ -64,11 +64,12 @@ class GpuTrack : public Track {
   [[nodiscard]] std::string GetTooltip() const override;
   [[nodiscard]] float GetHeight() const override;
 
+  [[nodiscard]] virtual std::vector<CaptureViewElement*> GetChildren() const override;
```

Comment:
Maybe the keyword `virtual` is not necessary here. And the same in the following places.

Reply:
Right, that's not needed, I'll remove this, thanks!